jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -362,6 +363,20 @@ class EndpointK @JvmOverloads constructor(
         }
     }
 
+    override fun getDebugState(): JSONObject {
+        return super.getDebugState().apply {
+            // debugState.put("sctpManager", sctpManager.getDebugState());
+            put("bitrateController", bitrateController.debugState)
+            put("bandwidthProbing", bandwidthProbing.debugState)
+            put("iceTransport", iceTransport.getDebugState())
+            put("dtlsTransport", dtlsTransport.getDebugState())
```

Comment:
I see we're also inconsistent between `val debugState` and `fun getDebugState()`; we should regularize these too, but not for this PR.

Reply:
Yeah, agreed.